libs/CCAppCommon/include/ccApplicationBase.h

```diff
@@ -23,16 +23,33 @@
 
 //! Mimic Qt's qApp for easy access to the application instance
 #define ccApp (static_cast<ccApplicationBase *>( QCoreApplication::instance() ))
-
+//! ccApplicationBase
+/*!
+ * ccApplicationBase provides a QApplication class with additional Constructor param isCommandLine and version
```

Comment:
Looks a little bit too detailed to me. For this kind of class without much intelligence, I don't think we always need a detailed description.